libs/styles/rounded-corner.css

```diff
@@ -0,0 +1,29 @@
+:root {
+  /* border-radius sizes */
+  --s-rounded-corners: 4px;
+  --m-rounded-corners: 8px;
+  --l-rounded-corners: 16px;
+  --full-rounded-corners: 50%;
+}
+
+.s-rounded-corners,
+.s-rounded-corners > .background img,
+.s-image-rounded-corners .foreground .image picture img {
```

Comment:
Few thoughts.. 

nit: I like the name `border-radius` better for the file name, css class names and vars. Authoring may prefer `*-rounded-corners` however, so disregard if that's been discussed otherwise Id lean towards css prop names if in doubt. 
ex. 
```
  --s-border-radius: 4px;
  --m-border-radius: 8px;
  --l-border-radius: 16px;
  --50-border-radius: 50%;
  ```
I do think we should update the variant 
  `*-image-rounded-corners` to... 
  `*-rounded-corners-image` 
to keep the names unified and have an optional type convention when incrementing utility css options `${unit-property-type}`

Reply:
Also, Is there a default size for just `rounded-corners` || `border-radius`?

Reply:
@ryanmparrish [See this feedback](https://github.com/adobecom/milo/pull/1433#issuecomment-1781579239) related to the naming

Reply:
The naming was done based on Chris's suggestion. Is there a need to still change it?

Reply:
I guess it's OK. I still hate it for the record. 
If we're talking about readability, When I read `s-image...` I immediately think image size, not what is the border radius of an image. Hence the suggestion `s-border-radius-image` etc. Anyhow thats my 2 cents.